logstash-core/spec/support/mocks_classes.rb

```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+require "logstash/outputs/base"
+
+class DummyOutput < LogStash::Outputs::Base
```

Comment:
`<not for this change, but ....>`
I'm now really seeing _too much_ mocks/dummy/whatever kind of plugins, is time to wipe all the real plugin dependencies and organize all this mocks properly.